docs/source/about/setup.rst

```diff
@@ -134,7 +134,7 @@ Install
 
          .. code-block:: bash
 
-            sudo apt install -f ./sunshine-{distro}-{distro-version}-{arch}.deb
+            sudo dpkg -i ./sunshine-{distro}-{distro-version}-{arch}.deb
```

Comment:
What's the advantage of `dpkg`?

We're using `apt` in our dockerfiles, e.g. https://github.com/LizardByte/Sunshine/blob/aadd6eec19f5fc1027e530a017b672409b31de1e/docker/ubuntu-22.04.dockerfile#L162

Although, I do think the `-f` arg is incorrect.

Reply:
It came up on Discord: https://discord.com/channels/804382334370578482/1256219495467454585/1256225884860321842
(apt seems to have some sandboxing that can lead to unexpected errors.)